electron/renderer/src/components/Webviews.js

```diff
@@ -29,10 +29,12 @@ class Webviews extends Component {
     super(props)
   }
 
-  _getEnvironmentUrl() {
+  _getEnvironmentUrl(account) {
     const url = new URL(window.location)
     const env = url.searchParams.get('env')
-    return decodeURIComponent(env)
+    const envUrl = new URL(env)
+    envUrl.searchParams.set('id', account.id)
+    return decodeURIComponent(envUrl)
   }
```

Comment:
in order to seperate the log files, i pass the the account id as a parameter to the webview